wallet-api/src/Ledger/Types.hs

```diff
@@ -29,8 +29,8 @@ module Ledger.Types(
     pubKeyAddress,
     scriptAddress,
     -- ** Scripts
-    Script, -- abstract
-    fromCompiledCode,
+    Script(..),
```

Comment:
Why make this no longer abstract?

Reply:
I wanted `compileScript` to replace `fromCompiledCode :: CompiledCode a -> Script`, but now there is no way to turn a `CompiledCode a` into a `Script`. (I suppose we could export both `compileScript` and `fromCompiledCode`)

Reply:
ok, `Script` is abstract again and `fromCompiledCode` is back.

Reply:
I'd prefer to do that. I think keeping `Script` abstract is just a good reminder to us that it's one of our interface pieces and should be as opaques as possible.

Reply:
I'm generally of the opinion that every constructor should be exposed somewhere, but I agree about keeping the interface well defined and adding a `Ledger.Types.Internal` module just for that is not worth it right now

Reply:
We can have that discussion another time ;)